cumulusci/core/tests/test_tasks.py

```diff
@@ -68,3 +93,45 @@ def _run_task(self):
         task()
 
         self.assertEqual(task.result, -1)
+
+    def test_task_logs_name(self):
+        """ A task logs the task class name to info (and not creds) """
+
+        task = _TaskHasResult(
+            self.project_config,
+            self.task_config,
+            self.org_config
+        )
+        task()
+
+        self.assertTrue(any(
+            "_TaskHasResult" in s for s in self.task_log['info']
+        ))
+
+    def test_salesforce_task_logs_org_id(self):
+        """ A salesforce_task will also log the org id & username """
+
+        task = _SfdcTask(
+            self.project_config,
+            self.task_config,
+            self.org_config
+        )
+        task()
+
+        self.assertTrue(any(
+            "00D000000000001" in s for s in self.task_log['info']
+        ))
+
+    def test_no_id_if_run_from_flow(self):
+        """ A salesforce_task will not log the org id if run from a flow """
+
+        task = _SfdcTask(
+            self.project_config,
+            self.task_config,
+            self.org_config,
+            flow=BaseFlow(self.project_config, FlowConfig(), self.org_config)
+        )
+        task()
+        self.assertFalse(any(
```

Comment:
now 

`    def test_task_logs_name_not_org(self):
        """ A task logs the task class name to info (and not creds) """
`